drracket-core-lib/drracket/private/syncheck/gui.rkt

```diff
@@ -2132,9 +2132,8 @@ If the namespace does not, they are colored the unbound color.
         (define/public (syncheck:clear-error-message)
           (define old-error-report-visible? error-report-visible?)
           (turn-off-error-report)
-          (when old-error-report-visible?
-            (when (is-current-tab?)
-              (send (get-frame) hide-error-report))))
+          (when (and old-error-report-visible? (is-current-tab?))
+            (send (get-frame) hide-error-report)))
         
```

Comment:
I merged one of these before, but I'm not convinced of this one. The indentation savings in this example isn't serious but the `and` makes it a bit harder to read, especially since it is all on one line.

Reply:
Interesting. I find this kind of thing more readable because it's easier to check that the code following it is outside the `when`. But maybe there's some room for nuance about when the rule is worth applying. I'll give it some thought. Let me know if you see other instances of this rule that seem off to you.